code/data_preprocessing/covariate/BiCV_factor.ipynb

```diff
@@ -96,6 +96,7 @@
     "parameter: numThreads = 8\n",
     "# Software container option\n",
     "parameter: container = \"\"\n",
+    "parameter: container_bioinfo = container\n",
```

Comment:
It doesn't really make sense to have to use two containers.

Reply:
If we dont use two containers, there are two other ways:
1. Add the bgzip and tabix to apex
2. Run the fake vcf as a explicit step.
I wonder which is better?

Reply:
I would go for the first proposal.